internal/storage/minio_chunk_manager.go

```diff
@@ -72,9 +72,9 @@ func newMinioChunkManagerWithConfig(ctx context.Context, c *config) (*MinioChunk
 			return err
 		}
 		if !bucketExists {
-			log.Debug("minio chunk manager new minio client", zap.Any("Check bucket", "bucket not exist"))
+			log.Info("minio chunk manager new minio client", zap.Any("Check bucket", "bucket not exist"))
```

Comment:
Confusing about this log message. I guess that bucket name should be printed here.